lib/mayaUsd/render/vp2RenderDelegate/draw_item.cpp

```diff
@@ -61,11 +61,9 @@ HdVP2DrawItem::AddRenderItem(MHWRender::MRenderItem* item, const HdGeomSubset* g
     RenderItemData& renderItemData = _renderItems.back();
 
     renderItemData._renderItem = item;
-    renderItemData._renderItemName = _drawItemName;
+    renderItemData._renderItemName = item->name();
```

Comment:
The name stored in the _renderItemData must exactly match the name of the MRenderItem. We use the name to remove the item from the subscene container. If the names do not match we'll leak items into the container and forget about them.

Reply:
Actually I changed the code a bit more so that we don't rely on _renderItemName. That member is meant for debugging only.